Require.sublime-settings

```diff
@@ -40,6 +40,9 @@
     // Use 'single' or "double" quotes
     "quotes": "single",
 
+    // Use 'var' or 'let' variable declarations
+    "var": "let",
```

Comment:
Whoops. I set `let` to default when testing to make sure it worked but forgot to set it back to var.